src/ROUTES.js

```diff
@@ -27,6 +27,8 @@ export default {
     SETTINGS_TIMEZONE_SELECT: 'settings/profile/timezone/select',
     SETTINGS_PRONOUNS: 'settings/profile/pronouns',
     SETTINGS_PREFERENCES: 'settings/preferences',
+    SETTINGS_PRIORITY_MODES: 'settings/preferences/priority-modes',
+    SETTINGS_LANGUAGES: 'settings/preferences/languages',
```

Comment:
I agree with singular for both of these. For naming conventions, I think we should focus on what the user is setting. i.e. they're setting their language, not their languages.

This rule of thumb would also lead us to use plural for pronouns (because each user sets multiple pronouns).

Reply:
I might be crazy but it looks like these are still plural?

Reply:
Aah it looks like pretty much everything else was changed to singular, just missed this little part

Reply:
oh yeah,  missed those. Sorry. They are fixed now.